app/components/event/event_component.rb

```diff
@@ -82,6 +82,10 @@ def primary_neighbourhood?
     event.neighbourhood == primary_neighbourhood || primary_neighbourhood.children.include?(event.neighbourhood)
   end
 
+  def online?
```

Comment:
is_online?

Reply:
doesn't fit with `repeats`, etc. the `is_` is redundant I think?

Reply:
its just a convention for a boolean to have is_ or has_ at the start i think?